ignite/config/plugins/config.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 type Config struct {
-	// path to the config file
 	path string
 
+	// Plugins holds the list of installed plugins.
 	Plugins []Plugin `yaml:"plugins"`
```

Comment:
```suggestion
	Apps []App `yaml:"plugins"`
```

Reply:
This change is implemented in #3669, which deals with the config file related changes